terraform/modules/front_door/ip_list.tf

```diff
@@ -2,10 +2,10 @@ locals {
   # This is used when restrict_by_ip is set to true, to restrict access to testing environments
   ip_allowlist = [
     # Softwire
-    "31.221.86.178",
-    "167.98.33.82",
-    "82.163.115.98",
-    "87.224.105.250",
+    "31.221.86.178/32",
+    "167.98.33.82/32",
+    "82.163.115.98/32",
+    "87.224.105.250/32",
```

Comment:
If we have Cambridge and Manchester do we also want to add Bucharest here? (I found the rest of the above here https://softwiretech.atlassian.net/wiki/spaces/Helpdesk/pages/6856573750/Network+WAN+Administration.)